Updates grpc-web-client

the729 opened the pull request:
Updates grpc-web-client to @improbable-eng/grpc-web.
Also fixes webpack entry file. Should be "dist/grpc-web-client.js" instead of "dist/index.js".

Updates #41

the729 committed:
Rename grpc-web-client to @improbable-eng/grpc-web, fix webpack entry

johanbrandhorst commented:
Yeah so the tests are failing because of a behaviour change in newer versions of grpc-web-client. While I'd ideally see these tests fixed, I'd be OK with just removing some of the checks. Could you go ahead and do that? The tests should be runnable locally with `make integration`.

If you find yourself trying to get stuff running and not getting anywhere, I'm afraid to say that's basically the state of this library at the moment. I don't have the will or energy to maintain it when GopherJS is falling behind the main distribution already.

the729 committed:
fix tests.

Do not expect anything about headers and trailers CallOptions on non-unary calls.

the729 commented:
tests fixed.

johanbrandhorst commented:
Thanks for your contribution!
